pf4j/src/main/java/org/pf4j/ManifestPluginDescriptorFinder.java

```diff
@@ -125,8 +125,8 @@ protected PluginDescriptor createPluginDescriptor(Manifest manifest) {
         return pluginDescriptor;
     }
 
-    protected PluginDescriptor createPluginDescriptorInstance() {
-        return new PluginDescriptor();
+    protected DefaultPluginDescriptor createPluginDescriptorInstance() {
```

Comment:
No need to change. I feel that it's from the refactoring action

Reply:
I think the refactoring is correct--I didn't include the mutators in the interface--do you want them there?  My objection is that it would prevent PluginDescriptor from ever becoming immutable.

Reply:
My bad. On the returned object we call setXYZ.